t/tars/discovery/tars.t

```diff
@@ -0,0 +1,381 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+use t::APISIX 'no_plan';
+
+repeat_each(1);
+log_level('warn');
+no_root_location();
+no_shuffle();
+workers(4);
+
+add_block_preprocessor(sub {
+    my ($block) = @_;
+
+    my $yaml_config = <<_EOC_;
+apisix:
+  node_listen: 1984
+  config_center: yaml
+  enable_admin: false
+discovery:
+  tars:
+    db_conf:
+      host: 127.0.0.1
+      port: 3306
+      database: db_tars
+      user: root
+      password: tars2022
+    full_fetch_interval: 90
```

Comment:
Could we use a shorter interval instead of waiting > 90 in the test?

Reply:
this value Is check by schema ,
can we use an fake schema.lua  int test ?

Reply:
Maybe we can reduce the min in the schema? A sane user may not use a low value.

Reply:
> A sane user may not use a low value.
Yes, I agree with this

But I think the meaning of scheame's existence is to ensure that this value is within a reasonable range,

Is it not appropriate to modify the schema if it is only for testing

Reply:
What about rewriting schema in the test?
Like https://github.com/apache/apisix/blob/9d450d7fe3169a77727df28696d083809b93977a/t/plugin/skywalking.t#L34-L35

Reply:
ok, I will try